whois-commons/src/main/resources/patch/whois-1.103.1.sql

```diff
@@ -0,0 +1,3 @@
+-- delete tags functionality from whois
+
+DROP TABLE IF EXISTS `tags`;
```

Comment:
The *current* whois release is 1.103, please use the *next* Whois release 1.104 for the patch file

Reply:
renamed to 1.104